spec/models/invitation_spec.rb

```diff
@@ -75,4 +75,41 @@
       end
     end
   end
+
+  describe "can_resend?" do
+    # Support old invites when AO3-6094 wasn't fixed.
+    context "old invites without sent_at" do
```

Comment:
[Context](https://www.betterspecs.org/#contexts) should start with `when` or `with/out`, so let's go with `without sent_at` here and `with sent_at` below. (I was going to say `without sent_at due to AO3-6094`, but there might be other reasons for a nil `sent_at`)